src/utils/scroll.ts

```diff
@@ -0,0 +1,17 @@
+export const scrollToID = (input: string, skipScroll: boolean = false) => (event: React.MouseEvent) => {
+    const id = input.split('#')[1] ?? input;
```

Comment:
This will strip a leading hash if it exists
```suggestion
    const id = id.replace(/^#/, '');
```